test/hotspot/jtreg/runtime/os/TestMemoryAllocationLogging.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
An Oracle copyright is not required in this new file as Oracle did not create it. But for reference note that Oracle copyrights require a comma after the year.